deploy/02_DeployStrategy.ts

```diff
@@ -0,0 +1,58 @@
+import { getGlobalDeploys, getMarketDeploys } from '@lyrafinance/protocol';
+import { ZERO_ADDRESS } from '@lyrafinance/protocol/dist/scripts/util/web3utils';
+import { DeployFunction } from 'hardhat-deploy/types';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+import { loadParams } from '../scripts/utils';
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre;
+  const { deploy } = deployments;
+  const { deployer } = await getNamedAccounts();
+  const lyraVault = await deployments.get('LyraVault');
+  const params = loadParams();
+
+  // get lyra addresses
+  const lyraGlobal = getGlobalDeploys(params.network);
+
+  await deploy(params.contract, {
+    from: deployer,
+    args: [lyraVault.address, params.optionType],
+    libraries: {
+      BlackScholes: lyraGlobal.BlackScholes.address,
+    },
+    log: true,
+  });
+
+  const lyraMarket = getMarketDeploys(params.network, params.vault.market);
+
+  // init Lyra Adapter
+  await deployments.execute(
+    params.contract,
+    {
+      from: deployer,
+      log: true,
+    },
+    'initAdapter',
+    lyraGlobal.LyraRegistry.address,
+    lyraMarket.OptionMarket.address,
+    ZERO_ADDRESS, // curve swap
+    ZERO_ADDRESS, // basic fee counter (not yet deployed by lyra)
```

Comment:
are these addresses available as variables in lyra sdk yet? if so better use them, if not we should create issue or add a @todo tag so we remember to update them before mainnet deployment

Reply:
Good catch - will add this to a todo in the lyraSDK. The fee counter hasn't been released yet so will add a @todo tag to this one.